chipsec/helper/win/win32helper.py

```diff
@@ -723,8 +723,9 @@ def get_descriptor_table( self, cpu_thread_id, desc_table_code  ):
     # EFI Variable API
     #
     def EFI_supported( self):
-        # @TODO: use GetFirmwareType ?
-        return ((self.GetFirmwareEnvironmentVariable is not None) or (self.GetFirmwareEnvironmentVariableEx is not None))
+        # kern32.GetFirmwareEnvironmentVariable with garbage parameters will return error = 1 reliably on a legacy system
+        self.GetFirmwareEnvironmentVariable("","{00000000-0000-0000-0000-000000000000}",0,0)
```

Comment:
Need to make sure the interface is not None prior to calling.  Looks like it may be None in some cases.